src/webgpu/api/validation/encoding/cmds/render/draw.spec.ts

```diff
@@ -723,3 +723,26 @@ In this test we test that only the last setting for a buffer slot take account.
 `
   )
   .unimplemented();
+
+g.test(`max_draw_count`)
+  .desc(
+    `
+In this test we test that draw count which exceeds
+GPURenderPassDescriptor.maxDrawCount causes validation error on
+GPUCommandEncoder.finish(). The test sets specified maxDrawCount,
+calls specified draw call specified times with or without bundles,
+and checks whether GPUCommandEncoder.finish() causes a validation error.
+    - x= whether use bundles for first/second draw calls
+    - x= several different draw counts
+    - x= several different maxDrawCounts
+`
+  )
+  .params(u =>
+    u
+      .combine('bundleFirstHalf', [false, true])
+      .combine('bundleSecondHalf', [false, true])
+      .combine('maxDrawCount', [0, 1, 4, 16])
+      .beginSubcases()
+      .expand('drawCount', p => [p.maxDrawCount, p.maxDrawCount + 1])
```

Comment:
```suggestion
      .expand('drawCount', p => [0, p.maxDrawCount, p.maxDrawCount + 1])
```

I'll go ahead and add this now, if we find that the test is slower than we want we can always remove it (I think most likely other tests somewhere in the CTS would give us coverage of anything that could possibly go wrong in the drawCount=0 case).